docs/readthedocs/source/doc/LLM/Quickstart/install_linux_gpu.md

```diff
@@ -0,0 +1,178 @@
+# Install BigDL-LLM on Linux with Intel GPU
+
+This guide demonstrates how to install BigDL-LLM on Linux with Intel GPUs. It applies to Intel Data Center GPU Flex Series and Max Series, as well as Intel Arc Series GPU.
+
+BigDL-LLM currently supports the Ubuntu 20.04 operating system and later, and supports PyTorch 2.0 and PyTorch 2.1 on Linux. This example installs BigDL-LLM with PyTorch 2.1 using `pip`. For more details and other options like installing with wheel, please refer to the [Installation Webpage](https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html#linux).
+
+
+## Install Intel GPU Driver
+Install Intel GPU Driver version >= stable_775_20_20231219. We highly recommend installing the latest version of intel-i915-dkms using apt.
+
+  > Note: Please refer to our [driver installation](install_linux_driver.md) detailed instruction in driver installation.
+
+## Setup Python Environment
+
+* Install the Miniconda as follows
+  ```bash
+  wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  bash Miniconda3-latest-Linux-x86_64.sh
+  source ~/.bashrc
+  # Verify the installation
+  conda --version
+  # rm Miniconda3-latest-Linux-x86_64.sh # if you don't need this file any longer
+  ```
+  > <img src="https://llm-assets.readthedocs.io/en/latest/_images/pythonenv_install_conda1.png" alt="image-20240221102252569" width=100%; />
+
+  > <img src="https://llm-assets.readthedocs.io/en/latest/_images/pythonenv_install_conda2.png" alt="image-20240221102252568" width=100%; />
+
+  > <img src="https://llm-assets.readthedocs.io/en/latest/_images/pythonenv_install_conda3.png" alt="image-20240221102252567" width=100%; />
+
+* Update and install packages
+  ```
+  conda update -n base conda -y
+  conda clean --all --yes
+  conda install pip -y
+  ```
+  > <img src="https://llm-assets.readthedocs.io/en/latest/_images/pythonenv_install_conda4.png" alt="image-20240221102252566" width=100%; />
+
+
+
+## Install oneAPI 
+  ```
+  wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB | gpg --dearmor | sudo tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null
+  echo "deb [signed-by=/usr/share/keyrings/oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main" | sudo tee /etc/apt/sources.list.d/oneAPI.list
+  sudo apt update
+
+  sudo apt install intel-basekit
+  ```
+  > <img src="https://llm-assets.readthedocs.io/en/latest/_images/install_oneapi_1.png" alt="image-20240221102252565" width=100%; />
+
+
+## Install `bigdl-llm`
+
+* With the `llm` environment active, use `pip` to install `bigdl-llm` for GPU:
+  ```
+  conda create -n llm python=3.9
+  conda activate llm
+
+  pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
```

Comment:
Shall we use `--extra-index-url`, similar to windows? @hkvision @Oscilloscope98

Reply:
Seems in the installation page we always use -f: https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html
I think both works, which one do you prefer and we can change all of them?

Reply:
> Seems in the installation page we always use `-f`: https://bigdl.readthedocs.io/en/latest/doc/LLM/Overview/install_gpu.html I think both works, which one do you prefer and we can change all of them?

I think `--extra-index-url` is preferred according to the IPEX team

Reply:
Have some verification that --extra-index-url work for Linux installation (verified on Arc21)